.github/copilot-instructions.md

```
@@ -43,8 +43,10 @@ globs: *
 - Convert &#96;require()&#96; statements to ES module format (&#96;import&#96;) or exclude script files from ESLint in &#96;eslint.config.mjs&#96;. Scripts using &#96;require()&#96; have been excluded from ESLint.
 - Remove unused exports or add them to the &#96;ignore&#96; list in &#96;knip.json&#96; if planned for future use.
 - Remove all unused variables identified by ESLint warnings.
-- Resolve type errors and linting errors to remove &#96;ignoreBuildErrors&#96; and &#96;ignoreDuringBuilds&#96; flags.
+- Resolve type errors and linting errors to remove &#96;ignoreBuildErrors&#96; and &#96;ignoreDuringBuilds&#96;.
 - Refactor data modules (&#96;akyo-data.ts&#96;, &#96;akyo-data-server.ts&#96;, &#96;akyo-data-json.ts&#96;, &#96;akyo-data-kv.ts&#96;) to reduce redundant logic, especially for functions like &#96;getAllCategories&#96; and &#96;getAllAuthors&#96;.
+- Refactor category update scripts and fix Onion bug:
+    - Create akyo-data-helpers.ts with common extractCategories, extractAuthors, findAkyoById
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)